gensim/models/ldamodel.py

```diff
@@ -84,7 +84,7 @@ def update_dir_prior(prior, N, logphat, rho):
 
     dprior = -(gradf - b) / q
 
-    if all(rho * dprior + prior > 0):
+    if (rho * dprior + prior > 0).all():
```

Comment:
The old way has never worked for me. Whenever I install Gensim from source I have to change this line. Can we just keep it this way?